guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -113,15 +113,19 @@ endif::[]
 This can be set by the `host_packages` parameter.
 . Optional: Select the *Update packages* option to update all packages on the host upon registration.
 This can be set by the `host_update_packages` parameter.
-. Optional: In the *Repository* field, enter a repository to be added before the registration is performed.
+. In the *Repositories* field, click the *+* symbol.
++
+On the *Repository list* window, add content that is required before performing the registration.
 For example, it can be useful to make the `subscription-manager` package available for the purpose of the registration.
++
+* In the *Repository* field, enter a repository to be added before the registration is performed.
 ifdef::client-content-dnf[]
 For Red Hat family distributions, enter the URL of the repository, for example `\http://rpm.example.com/`.
 endif::[]
 ifdef::client-content-apt[]
 For Debian OS families, enter the whole line of list file content, for example `deb \http://deb.example.com/ bookworm 1.0`.
 endif::[]
-. Optional: In the *Repository GPG key URL* field, specify the public key to verify the signatures of GPG-signed packages.
+* Optional: In the *Repository GPG key URL* field, specify the public key to verify the signatures of GPG-signed packages.
 It needs to be specified in the ASCII form with the GPG public key header.
```

Comment:
```suggestion
It needs to be specified in the ASCII form with the GPG public key header.

+
You do not have to specify repositories if you provide them in an activation key.
```
I know this is out of PR scope and I'm not sure if my wording is true/correct, but I think that there are cases when the user doesn't have to specify any repos. It would be good to mention it.

Reply:
Applied.